src/app/tests/TestReadInteraction.cpp

```diff
@@ -93,7 +93,7 @@ void TestReadInteraction::TestReadClient(nlTestSuite * apSuite, void * apContext
     err                            = readClient.Init(&gExchangeManager, nullptr);
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
-    err = readClient.SendReadRequest(kTestDeviceNodeId, gAdminId, nullptr, 0);
+    err = readClient.SendReadRequest(kTestDeviceNodeId, gAdminId, nullptr, 0, nullptr, 0);
```

Comment:
add comments on what the parameter mean. nullptr and 0 are not obvious.

This seems to highlight the issue of having too many arguments to this method.

Reply:
Done, thanks, just update comments, would create another ticket to optimize send read request function aguments, thanks